packages/web-application/src/web-application.ts

```diff
@@ -901,16 +648,20 @@ export class WebApplication extends PluggableModule {
             }
 
             try {
-                let element = getElementByXPath(xpath);
-                let evt = document.createEvent('HTMLEvents');
+                const element = getElementByXPath(xpath);
 
                 if (element) {
-                    element.focus();
                     element.value = value;
+                    const eventInit = {
```

Comment:
Only bubbles argument could break the code.
Maybe take a snapshot of real even, cause not every one is checking that keyCode or char is undefined.
In that case when we will trigger this event, the js code that doesn't check this signarute will fail with error